compass/landice/tests/thwaites/__init__.py

```diff
@@ -1,7 +1,7 @@
 from compass.testgroup import TestGroup
 from compass.landice.tests.thwaites.decomposition_test import DecompositionTest
 from compass.landice.tests.thwaites.restart_test import RestartTest
-
+from compass.landice.tests.thwaites.high_res_mesh import HighResMesh
```

Comment:
```suggestion
from compass.landice.tests.thwaites.high_res_mesh import HighResMesh

```
Two blank lines after imports.